dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenAPI-2.0/io.openliberty.mpOpenAPI-2.0.feature

```diff
@@ -0,0 +1,54 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpOpenAPI-2.0
+visibility=public
+singleton=true
+IBM-App-ForceRestart: install, \
+  uninstall
+IBM-ShortName: mpOpenAPI-2.0
+Subsystem-Name: MicroProfile OpenAPI 2.0
+IBM-API-Package: \
+    org.eclipse.microprofile.openapi.annotations;  type="stable",\
+	org.eclipse.microprofile.openapi.annotations.callbacks; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.enums; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.extensions; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.headers; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.info; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.links; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.media; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.parameters; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.responses; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.security; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.servers; type="stable",\
+	org.eclipse.microprofile.openapi.annotations.tags; type="stable",\
+	org.eclipse.microprofile.openapi; type="stable",\
+	org.eclipse.microprofile.openapi.models; type="stable",\
+	org.eclipse.microprofile.openapi.models.callbacks; type="stable",\
+	org.eclipse.microprofile.openapi.models.examples; type="stable",\
+	org.eclipse.microprofile.openapi.models.headers; type="stable",\
+	org.eclipse.microprofile.openapi.models.info; type="stable",\
+	org.eclipse.microprofile.openapi.models.links; type="stable",\
+	org.eclipse.microprofile.openapi.models.media; type="stable",\
+	org.eclipse.microprofile.openapi.models.parameters; type="stable",\
+	org.eclipse.microprofile.openapi.models.responses; type="stable",\
+	org.eclipse.microprofile.openapi.models.security; type="stable",\
+	org.eclipse.microprofile.openapi.models.servers; type="stable",\
+	org.eclipse.microprofile.openapi.models.tags; type="stable"
+IBM-SPI-Package: \
+    org.eclipse.microprofile.openapi.spi; type="stable"
+-features=\
+ io.openliberty.org.eclipse.microprofile.openapi-2.0, \
+ com.ibm.websphere.appserver.servlet-3.1; ibm.tolerates:=4.0, \
+ com.ibm.websphere.appserver.mpConfig-1.3; ibm.tolerates:="1.4", \
+ com.ibm.websphere.appserver.internal.optional.jaxb-2.2; ibm.tolerates:=2.3, \
```

Comment:
Oooh, what's this internal optional jaxb feature? Never seen it before.

Reply:
don't do the ibm:tolerates. This should just depend on mpConfig-2.0 when it is ready.  don't tolerate to servlet-4.0